test/feed-worker-filtertest.js

```diff
@@ -0,0 +1,14 @@
+const feedQueue = require('../src/feed-queue');
```

Comment:
This isn't really a test.  Did you want to actually write some tests using Jest for this code and add to `test/` or file an Issue for someone to do it later?

Reply:
@humphd @jerryshueh 

I can create a tester for this.

Reply:
Looks like it can be a good issue for people who want to jump in on making test. Let's post this formally after this PR gets passed.